src/utils/NetworkUtils.ts

```diff
@@ -1,5 +1,8 @@
 import { PublicNetworks } from "@uma/common/dist/PublicNetworks";
 
+export const productionChainIds = [1, 10, 137, 324, 8453, 42161];
+export const testnetChainIds = [5, 280, 420, 80001, 84531, 421613];
```

Comment:
Thoughts on adding this to `constants.ts` since it's very general and could be useful elsewhere?

Reply:
Makes sense - good call :+1: https://github.com/across-protocol/sdk-v2/pull/461/commits/768139211d72a27ac4827cf10181106356088357